docs/wp_rigid_body_physics.rst

```diff
@@ -680,7 +680,9 @@ Here is our concrete schema proposal in full:
        doc = """Applies physics body attributes to any UsdGeomXformable prim and
        marks that prim to be driven by a simulation. If a simulation is running
        it will update this prim's pose. All prims in the hierarchy below this 
-       prim should move accordingly."""
```

Comment:
I'm not entirely sure what this file is for. I updated the schema docs here just because of search/replace really. I noticed it also has some out-of-date information that presumably came from overview.dox at one time as well, but I haven't updated that. Is there some automated process to synchronize them?

Reply:
I dropped the changes from this file (rebased into the docs commit)